backend/Bindings/Edna.Bindings.LtiAdvantage/Utils/EdnaExternalHttpHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Net.Http;
```

Comment:
Any specific reason for keeping this entry tied to `LtiAdvantage` - even though Lti1 is also using it? Maybe we can move it to general utils instead 🤔.

Reply:
I have one implementation in the general utils project as well, which is being used by the Lti1 binding, and the learn-content-api. Adding a dependency for Utils to LtiAdvantageBinding was causing a version conflict. Regarding that, I believe that we should take that (+ refactoring and removing this redundancy) as a separate work item, so that if things break apart, we are able to identify the cause- if it was the version upgrade, or the certificate validation changes. Any thoughts on this?